typegate/native/Cargo.toml

```diff
@@ -50,7 +48,7 @@ rust-s3 = "0.33.0"
 http = "0.2.9"
 envconfig = "0.10.0"
 once_cell = "1.18.0"
-wasmedge-sdk = { version = "0.8.1", features = ["standalone"] }
+wasmedge-sdk = { version = "0.8.1" }
```

Comment:
an issue with that? #360 is testing the removal of the flag but needs additional changes

Reply:
I have a linking error for the typegate/native on another machine...

Reply:
OK, I will remove it in the other pull request but it will be needed here for the test